config/locales/ar/rails.yml

```diff
@@ -1,3 +1,15 @@
+#Files in the config/locales directory are used for internationalization
+#and are automatically loaded by Rails. If you want to use locales other
+#than English, add the necessary files in this directory.
+#To use the locales, use `I18n.t`:
+#I18n.t "hello"
+#In views, this is aliased to just `t`:
+#<%= t("hello") %>
+#To use a different locale, set it with `I18n.locale`:
+#I18n.locale = :es
+#This would use the information in config/locales/es.yml.
+#To learn more, please read the Rails Internationalization guide
+#available at http://guides.rubyonrails.org/i18n.html.
```

Comment:
Any idea why these comments were added to all `rails.yml` files? The comments are misleading because they mention the `config/locales/es.yml` file, which we don't use; we use folders instead.

Reply:
Nice catch @javierm. I will ask to Crowdin support service!

Reply:
My guess was right, Crowdin added this behavior since September of this year, just because of that this is the first time we see this kind of changes in Crowdin commits. It is not optative so we have to accept those comments.

IMO, we can do 2 things to solve this:
* Remove the comment from main language. I guess Crowdin will remove them from all rails.yml files
* Update the comments of `config/locales/en/rails.yml` so at least those comments are correct

What do you think @javierm ?

Reply:
I vote for removing the comment in the main language file. To me it doesn't make much sense to have a comment in `en/rails.yml` and not in the other translation files :thinking:.

Reply:
Hi @javierm, finally i have created a MR #3887 to remove those comments from main language rails.yml file. I also found another one within `config/locales/en/devise.yml` with a comment but decide to keep it because i think this comment could be useful to someone.

Reply:
Cool :smile:. I've merged that pull request. It looks like crowdin has updated the files in the `i18n_master` but not on this pull request :thinking:.

Reply:
Yes this pull-request is manually managed by me and is not automatically updated when i18n_master branch change. I will push a new version with all those new changes from the i18n_master branch.

Reply:
Solved through #3887